.rwx/postmortem_report.py

```
@@ -0,0 +1,115 @@
+                    out[k] = v
+            break
+    return out
+
+
+def summarize_sessions(log_dir: Path) -> dict:
+    """Count sessions and success/fail from log contents."""
+    sessions = {}
+    for kind, sid, iid, path in find_session_logs(log_dir):
+        if kind == "session":
+            content = path.read_text(errors="replace")
+            success = "Result: SUCCESS" in content
```

Ruff (0.15.1)

[error] Local variable `success` is assigned to but never used

Remove assignment to unused variable `success`

(F841)